llvm/lib/AsmParser/LLParser.cpp

```diff
@@ -3135,9 +3135,21 @@ bool LLParser::parseOptionalOperandBundles(
         return true;
 
       Type *Ty = nullptr;
-      Value *Input = nullptr;
-      if (parseType(Ty) || parseValue(Ty, Input, PFS))
+      if (parseType(Ty))
         return true;
+
+      Value *Input = nullptr;
+      // FIXME: Metadata operand bundle value is garbage when LLVM IR is
+      // compiled to bitcode, then disassembled back to LLVM IR. See PR#89649
+      // for the reproducers, and https://bugs.llvm.org/show_bug.cgi?id=51264
```

Comment:
```suggestion
      // compiled to bitcode, then disassembled back to LLVM IR. 
      // See [PR#89649](https://github.com/llvm/llvm-project/pull/89649)
      // for the reproducers, and https://github.com/llvm/llvm-project/issues/50608 
```
I think URLs are better here, since developers can use them directly. The old bugzilla bug also has been migrated to github, so I think its better to use that link.

Reply:
I don't think it makes sense to only add support for one half of this... If we add IR parsing support, then bitcode writing should also work.